src/ServicePulse.Host/app/js/app.constants.js

```diff
@@ -5,7 +5,7 @@
         .constant('showPendingRetry', false)
         .constant('scConfig', {
             service_control_url: 'http://localhost:33333/api/',
-            monitoring_urls: ['http://localhost:33633/']
+            monitoring_urls: ['http://localhost:1234/']
```

Comment:
We can, but if I undo the monitoring server won't connect to SP, at least for me. Are you sure?